downstream/modules/platform/ref-system-requirements.adoc

```diff
@@ -30,12 +30,16 @@ h| Python | 3.8 or later |
 
 
 
-.Automation controller
+== Automation controller
+
+{ControllerNameStart} is a distributed system, where different software components can be colocated or distributed across multiple compute nodes.
```

Comment:
```suggestion
{ControllerNameStart} is a distributed system, where different software components can be co-located or deployed across multiple compute nodes.
```

Reply:
avoid duplication of "distributed" - seems like you're using the same word to describe itself. 

also not 100% but I feel like there should be a hyphen in `co-located` otherwise it looks like a misspelling of "collocated"